app/services/doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -114,16 +114,20 @@ def classification_info
           return unless classification_details.present?
 
           classification_details.transform_values do |classification_detail|
-            classification_detail.slice(
+            detail = classification_detail.slice(
               'ClassName',
               'Issue',
               'IssueType',
               'Name',
               'IssuerCode',
               'IssuerName',
               'CountryCode',
+              'IssuerType',
             )
-          end
+            # code to value
+            detail['IssuerType'] = DocAuth::Acuant::IssuerTypes.from_int(detail['IssuerType']).name
```

Comment:
Double-checking my understanding: so `IssuerType` is passed through as an integer?

If so, where did you find the definition for each number?

Reply:
@night-jellyfish https://github.com/Acuant/AndroidSDKV11/blob/master/docs/MigrationDetails.md.

Reply:
Awesome, thank you!